addons/silicon.util.custom_docs/doc_exporter/editor_help_doc_exporter.gd

```diff
@@ -0,0 +1,1046 @@
+			
+			if property.setter != "":
+				skip_methods.append(property.setter)
+				describe = true
+			if property.getter != "":
+				skip_methods.append(property.getter)
+				describe = true
+			
+			if property.description != "":
+				describe = true
+			
+			label.push_cell()
+			label.push_font(doc_code_font)
+			label.push_color(headline_color)
+			
+			if describe:
+				label.push_meta("@member " + property.name)
+			
+			add_text(property.name)
+			
+			if describe:
+				label.pop()
+				property_descr = true
+			
+			if property.default != "":
+				label.push_color(symbol_color)
+				label.add_text(" [default: ")
+				label.pop()
+				label.push_color(value_color)
+				add_text(property.default)
+				label.pop()
+				label.push_color(symbol_color)
+				label.add_text("]")
+				label.pop()
+
+			label.pop()
+			label.pop()
+			label.pop()
+
+		label.pop() #table
+		label.pop()
+		label.pop() # font
+		label.add_text("\n")
+		label.add_text("\n")
+
+	# Methods overview
+	var method_descr := false
+	var sort_methods: bool = editor_settings.get("text_editor/help/sort_functions_alphabetically")
+	var methods := []
+	
+	for method in doc.methods:
+		if skip_methods.has(method.name):
+			if method.args.size() == 0 or (method.args.size() == 1 and method.return_type == "void"):
+				continue
+		methods.append(method)
+	
+	if methods.size():
+		if sort_methods:
+			methods.sort_custom(self, "sort_methods")
+		if is_current:
+			section_lines.append(["Methods", label.get_line_count() - 2])
+		label.push_color(title_color)
+		label.push_font(doc_title_font)
+		label.add_text("Methods")
+		label.pop()
+		label.pop()
+		
+		label.add_text("\n")
+		label.push_font(doc_code_font)
+		label.push_indent(1)
+		label.push_table(2)
+		label.set_table_column_expand(1, true, 1)
+		
+		var any_previous := false
+		for _pass in 2:
+			var m := []
+			for method in methods:
+				if (_pass == 0 and method.is_virtual) or (_pass == 1 and not method.is_virtual):
+					m.append(method)
+			
+			if any_previous and not m.empty():
+				label.push_cell()
+				label.pop() #cell
+				label.push_cell()
+				label.pop() #cell
+			
+			var group_prefix := ""
+			for i in m.size():
+				var new_prefix: String = m[i].name.substr(0, 3)
+				var is_new_group := false
+				
+				if i < m.size() - 1 and new_prefix == m[i + 1].name.substr(0, 3) and new_prefix != group_prefix:
+					is_new_group = i > 0
+					group_prefix = new_prefix
+				elif group_prefix != "" and new_prefix != group_prefix:
+					is_new_group = true
+					group_prefix = ""
+				
+				if is_new_group and _pass == 1:
+					label.push_cell()
+					label.pop() #cell
+					label.push_cell()
+					label.pop() #cell
+				
+				if m[i].description != "":
+					method_descr = true
+				
+				add_method(m[i], true)
+			
+			any_previous = !m.empty()
+		
+		label.pop() #table
+		label.pop()
+		label.pop() # font
+		label.add_text("\n")
+		label.add_text("\n")
+
+	# Theme properties
+#	if doc.theme_properties.size():
```

Comment:
We should get rid of all the commented out code, if it's not going to be used.

Reply:
In terms of that plugin, I might expose theme properties in the future, so I'm just leaving the commented code there until I decide to implement it.